reaktive/src/commonMain/kotlin/com/badoo/reaktive/utils/Utils.kt

```diff
@@ -10,7 +12,8 @@ internal fun handleSourceError(error: Throwable, onError: ((Throwable) -> Unit)?
             } catch (e: Throwable) {
```

Comment:
Please, rename `e` into `errorHandlerException` or whatever. It is hard to distinct them.